Dynatrace/dynatrace-operator
# Skip generation of v1beta3 for release 1.3.0 and remove maxDescLen parameter

## Description

Make sure work-in-progress DynaKube v1beta3 is not release yet and does not have negative side-effects.

## How can this be tested?

* Deploy operator
* Deploy v1beta2 Dynakube -> should work
* Deploy v1beta3 Dynakube -> shall be rejected
